include/ensmallen_bits/ada_delta/ada_delta.hpp

```diff
@@ -128,6 +132,11 @@ class AdaDelta
   //! Modify whether or not the individual functions are shuffled.
   bool& Shuffle() { return optimizer.Shuffle(); }
 
+  //! Get whether or not the actual objective is calculated after training.
```

Comment:
Not sure, I would probably remove the "after training" part, see this more as an optimisation process.

Reply:
Sorry, I just got carried away.